pages/operate-icon/how-to-run-a-validator-node.mdx

```diff
@@ -1,12 +1,20 @@
 # How to run a Validator node
 
-### Prerequisites
+Registering and running a validator node requires a few steps to be followed.
 
-Install [Docker](https://www.docker.com/get-started/)
+* Create a wallet for the validator and register it on the chain as a validator by paying the registration fee.
+* Setup the node and wait for it to fully sync up. This node will be setup with a node wallet (not the same as validator wallet for security reasons) that will be used to sign blocks and consensus.
+* Register the node with the validator public key on the chain by calling `setPRepNodePublicKey` method.
```

Comment:
How about changing the sequence? Then users don't waste validator registration fees when users fail to set up a node.
1. create a wallet for the node and setup the node 
2. create a wallet for the validator and register on the chain
3. register the node public key

Reply:
completely agree, thanks for pointing that out.

I'll fix it and then publish 👍